src/ursa/util/memory_logger.py

```diff
@@ -32,10 +32,9 @@ def __init__(
         path: Optional[str | Path] = None,
         collection_name: str = "agent_memory",
     ) -> None:
+        home_dir = os.path.expanduser("~")
         self.path = (
-            Path(path)
-            if path
-            else Path(__file__).resolve().parent / "agent_memory_db"
+            Path(path) if path else Path(home_dir + "/.cache/ursa/rag/db/")
```

Comment:
```suggestion
            home_dir / ".cache" / "ursa" / "rag" / "db"
            if path is None
            else Path(path)
```

The reason for this is `if path` could give unexpected results if `path = ""`